icubam/predicu/data.py

```diff
@@ -123,6 +123,7 @@ def load_icubam(
       d = pd.read_csv(url.format(api_key))
   if preprocess:
     d = d.rename(columns={"create_date": "date", "icu_dept": "department"})
+    d.loc[d.department == "St-Dizier", "department"] = "Haute-Marne"
```

Comment:
Does it mean that the originally is not correct? Or is this information missing?

Reply:
yes. this entry in the db is incorrect.